src/core/Akka.Remote/Endpoint.cs

```diff
@@ -256,59 +263,71 @@ public OversizedPayloadException(string msg)
 
     /// <summary>
     /// INTERNAL API
-    /// 
-    /// <remarks>
```

Comment:
Comment was out of date - no longer true. We have reliable delivery inside the `ReliableDeliverySupervisor` now and have had it for quite some time :p